### PR TITLE
Support building and running a ballerina project

### PR DESCRIPTION
**Pull request by azinneera**

### CODE REVIEW

File: components/cli/cmd/cellery/build.go [hunk 1]
```diff
@@ -20,6 +20,10 @@ package main
 
 import (
 	"fmt"
+	"os"
+	"path/filepath"
+
+	"cellery.io/cellery/components/cli/pkg/constants"
```
Comment (MadhukaHarith92) [inline components/cli/cmd/cellery/build.go]: Code formatting issue.

File: components/cli/pkg/commands/image/testdata/project/build_artifacts/hr_reference.json [hunk 2]
```diff
@@ -0,0 +1 @@
+{"gateway_host":"{{instance_name}}--gateway-service","hr_hr_api_url":"http://{{instance_name}}--gateway-service:80/hr"}
```
Comment (MadhukaHarith92) [inline components/cli/pkg/commands/image/testdata/project/build_artifacts/hr_reference.json]: New line.

File: components/cli/pkg/commands/image/testdata/project/build_artifacts/foo_reference.json [hunk 3]
```diff
@@ -0,0 +1 @@
+{"gateway_host":"{{instance_name}}--gateway-service"}
```
Comment (MadhukaHarith92) [inline components/cli/pkg/commands/image/testdata/project/build_artifacts/foo_reference.json]: New line.

File: components/cli/pkg/commands/image/testdata/project/build_artifacts/hr_reference.json (hunk 2 above)
Comment (azinneera) [reply components/cli/pkg/commands/image/testdata/project/build_artifacts/hr_reference.json]: Fixed in 5296ee4

File: components/cli/pkg/commands/image/testdata/project/build_artifacts/foo_reference.json (hunk 3 above)
Comment (azinneera) [reply components/cli/pkg/commands/image/testdata/project/build_artifacts/foo_reference.json]: Fixed in 5296ee4

**Pull request merged**